test/erlcloud_s3_tests.erl

```diff
@@ -131,6 +132,15 @@ encode_bucket_lifecycle_tests(_) ->
     ?_assertEqual(Expected, Result),
     ?_assertEqual(Expected2, Result2).
 
+set_bucket_notification_test_() ->
+    [?_assertEqual({'NotificationConfiguration',[]},
+                   erlcloud_s3:create_notification_xml([])),
+     ?_assertEqual(?S3_BUCKET_EVENTS_SIMPLE_XML_FORM,
+                   erlcloud_s3:create_notification_xml(?S3_BUCKET_EVENTS_LIST))].
+
+get_bucket_notification_test_() ->
+    ?_assertEqual(?S3_BUCKET_EVENTS_LIST,
+                  erlcloud_s3:get_bucket_notification(?S3_BUCKET_EVENT_XML_CONFIG)).
```

Comment:
Let's start test this function. Probably some day somebody will add tests for other cases.